assignment/core/src/main/resources/OSGI-INF/config/CryptoConfiguration.stage.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
```

Comment:
Will this config get picked up for the appropriate environment ?
Is this up to the folder structure according to the project archetype

Reply:
changed the folder structure pleaser review changes